docs/indepth/installation.rst

```diff
@@ -374,6 +366,16 @@ Paste them into a root console on the target node:
 Post installation configuration for Fedora 20 and CentOS 7
 ----------------------------------------------------------
 
+First disable SELINUX.
+
+.. task:: disable_selinux
+   :prompt: [root@node]#
+
+.. note:: Flocker does not currently set the necessary SELINUX context types on the filesystem mountpoints that it creates on nodes.
+          This prevents Docker containers accessing those filesystems as volumes.
+          A future stable version of Flocker will set the appropriate SELINUX contexts.
```

Comment:
Ooh.  A forward-looking statement.  Quite bold of you.

I suggest making this less specific.  A statement like "A future version of Flocker may provide a different integration strategy." or something else weasel-y (on the basis that we don't actually know what we're going to do yet).